intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -296,6 +301,9 @@ model_token Model::post_process(float* logits) {
     } else {
       return post_sample_top_k_top_p_repeat(logits);
     }
+  } else {
+    fprintf(stderr, "error, beam search not supported!\n");
```

Comment:
beam search supported in generate_tokens

Reply:
updated in fdf5a88